config/default-spinnaker-local.yml

```diff
@@ -143,7 +146,10 @@ providers:
       username: ${OS_USERNAME}
       password: ${OS_PASSWORD}
       projectName: ${OS_PROJECT_NAME}
+      domainName: ${OS_PROJECT_DOMAIN_NAME}
```

Comment:
while you may have both, it seems overspecified to be using both and seems wrong to have one file overwrite the other. Pick one as your interface, or create an attribute for each and do the right thing internally to support that. I'm not familiar with open stack to tell you what the right thing is.

Keep in mind that spinnaker.yml will always be present. That is your base interface.
The default-spinnaker-local.yml is purely optional and is only used to default an initial spinnaker-local.yml if one was not explicitly supplied. So you cannot count on anything in it showing up in the runtime.